### PR TITLE
LPS-85097 Review toggle buttons for email and push notifications

### PR DESCRIPTION
**Pull request by dgarciasarai**

### CODE REVIEW

File: modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/notification/EmailNotificationSender.java [hunk 1]
```diff
@@ -111,16 +117,37 @@ protected void doSendNotification(
...
 			getInternetAddresses(
 				notificationRecipients.get(NotificationReceptionType.CC)));
+
 		mailMessage.setBCC(
 			getInternetAddresses(
 				notificationRecipients.get(NotificationReceptionType.BCC)));
 
+		List<InternetAddress> bulkAddresses = new ArrayList<>();
+
+		Collection<Set<NotificationRecipient>> notificationRecipientCollection =
```
[the remark this answers is not in the document]
Comment (dgarciasarai) [reply modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/notification/EmailNotificationSender.java]: I'll go with notificationRecipientSetCollection 👍

File: modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/notification/PushNotificationMessageSender.java [hunk 2]
```diff
@@ -115,9 +121,28 @@ protected void doSendNotification(
 			ExecutionContext executionContext)
 		throws Exception {
 
+		List<NotificationRecipient> recipientList = new ArrayList<>();
```
[the remark this answers is not in the document]
Comment (dgarciasarai) [reply modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/notification/PushNotificationMessageSender.java]: There is a parameter in the method with the same name, I'm doing a new list for those who will receive the notification

File: modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/notification/PushNotificationMessageSender.java [hunk 3]
```diff
@@ -115,9 +121,28 @@ protected void doSendNotification(
...
+
+		Collection<Set<NotificationRecipient>> notificationRecipientCollection =
```
[the remark this answers is not in the document]
Comment (dgarciasarai) [reply modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/notification/PushNotificationMessageSender.java]: The same as in EmailNotificationSender: notificationRecipientSetCollection

**Comment by nhpatt** [comment]
👍

**Pull request merged**